test/validation/api/dma/dma.c

```diff
@@ -452,6 +454,7 @@ static int do_transfer_async(odp_dma_t dma, const odp_dma_transfer_param_t *trs_
 		CU_ASSERT(done == 1);
 		CU_ASSERT(result.success);
 		CU_ASSERT(result.user_ptr == user_ptr);
+		CU_ASSERT(*(uint32_t *)result.user_ptr == USER_DATA);
```

Comment:
To avoid crash (and debugging), this assert could be executed only if the user pointer is correct.

Reply:
Fixed in v2.